test/tower_error_tracker_test.exs

```diff
@@ -219,16 +219,31 @@ defmodule TowerErrorTrackerTest do
     )
   end
 
-  test "Logger messages not reported because not supported by ErrorTracker" do
+  test "Logger messages reported as special custom exceptions (because messages not supported by ErrorTracker)" do
     in_unlinked_process(fn ->
       require Logger
 
       capture_log(fn ->
-        Logger.emergency("Panic!")
+        Logger.emergency("Emergency!")
+
+        Logger.critical("Critical!")
       end)
     end)
 
-    assert [] = TestApp.Repo.all(ErrorTracker.Error) |> TestApp.Repo.preload(:occurrences)
+    assert_eventually(
+      [
+        %{
+          kind: "message",
+          reason: "[emergency] Emergency!",
+          occurrences: [_]
+        },
+        %{
+          kind: "message",
+          reason: "[critical] Critical!",
+          occurrences: [_]
+        }
+      ] = TestApp.Repo.all(ErrorTracker.Error) |> TestApp.Repo.preload(:occurrences)
```

Comment:
This tests that both messages are treated as separate errors (with different fingerprints) and not different occurrences of the same error.